.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
```

Comment:
It seems that vscode team decided to change from boolean to strings.



Turns out, VSCode true is set to "explicit" since VSCode 1.85.0. It seems boolean values were supported until 1.84.0.

References

https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto https://code.visualstudio.com/updates/v1_83#_code-actions-on-save-and-auto-save